README.md

```diff
@@ -15,175 +15,10 @@ With [Managed Service Identity (MSI)](https://docs.microsoft.com/en-us/azure/app
 
 >Here's another sample that how to use MSI from inside an Azure VM with a Managed Service Identity (MSI) - [https://github.com/Azure-Samples/resource-manager-python-manage-resources-with-msi](https://github.com/Azure-Samples/resource-manager-python-manage-resources-with-msi)
```

Comment:
Change:  Here's another sample that demonstrates using a Managed Service Identity (MSI) from within an Azure VM.